### PR TITLE
Add onetrust-wrapper unit tests

### PR DESCRIPTION
**Pull request by silesky**
Testing is for winners. 🥇

### CODE REVIEW

File: packages/consent/consent-wrapper-onetrust/src/lib/__tests__/onetrust-api.test.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+beforeEach(() => {
+  // @ts-ignore
+  delete window.OneTrustActiveGroups
+  // @ts-ignore
+  delete window.OneTrust
+})
+
+describe(getConsentedGroupIds, () => {
+  it('should return an empty array if no groups are consented', () => {
```
Comment (danieljackins) [inline packages/consent/consent-wrapper-onetrust/src/lib/__tests__/onetrust-api.test.ts:13]: Is this test named incorrectly?

**Review by danieljackins — APPROVED** [review]
LGTM! Just one comment about a test name

File: packages/consent/consent-wrapper-onetrust/src/lib/__tests__/onetrust-api.test.ts (hunk 1 above)
Comment (silesky) [reply packages/consent/consent-wrapper-onetrust/src/lib/__tests__/onetrust-api.test.ts:13]: Hmm... it looks good to me eyes. What am I missing?

Comment (danieljackins) [reply packages/consent/consent-wrapper-onetrust/src/lib/__tests__/onetrust-api.test.ts:13]: I'm probably missing something, but the test says should return an empty array(...), and then we're expecting a non-empty array?
